WordPress/src/main/java/org/wordpress/android/ui/comments/CommentsListFragment.java

```diff
@@ -69,7 +76,7 @@
         SPAM(R.string.comment_status_spam),
         DELETE(R.string.comment_status_trash);
 
-        private final int mLabelResId;
+        public final int mLabelResId;
```

Comment:
Maybe we can consider keeping it private and adding a method like `getLabelResId` or even using the `getLabel` with the setText fun taking the string as input in `tab.setText(commentListFilters[position].mLabelResId)`, wdyt?